addons/code-analysis/style.css

```diff
@@ -0,0 +1,239 @@
+.ps-container {
+  position: relative;
+  background: #393943;
+  width: 960px;
+  height: 283px;
+  border-radius: 13px;
+}
+
+.ps-header {
+  position: absolute;
+  left: 15px;
+  top: 12px;
+  background: #ffffff;
+  width: 159px;
+  height: 44px;
+  border-radius: 13px;
+}
+
+.ps-header-right {
```

Comment:
This looks a little off. Maybe moving it down or reducing the padding a little would help?